lua/tardis/sh_screen_ui.lua

```diff
@@ -244,28 +244,37 @@ function TARDIS:RemoveHUDScreen()
     end
 end
 
-function TARDIS:HUDScreen()
+function TARDIS:HUDScreen(window)
     if IsValid(self.screenpopframe) then
         self:RemoveHUDScreen()
-        return
+        if not self.screen_in_context_menu then
+            return
+        end
+        self.screen_in_context_menu = nil
     end
 
-    local frame=vgui.Create("DFrame")
+    local frame = window or vgui.Create("DFrame")
     frame:SetSkin("TARDIS")
     frame:SetTitle(TARDIS:GetPhrase("Common.Interface"))
     frame:SetDraggable(true)
-    frame:ShowCloseButton(false)
-    frame:MakePopup()
+
+    if not window then
+        frame:ShowCloseButton(false)
+        frame:MakePopup()
+    end
+
     self.screenpopframe=frame
+    self.screen_in_context_menu = (window ~= nil)
 
     local screen = vgui.Create("DPanel",frame)
     screen.id="pop"
-    screen.width=825
+    screen.width=625
```

Comment:
Curious what the reason was for changing the width here? I thought I noticed it seems a little less wide in-game

Reply:
Was:
![image](https://user-images.githubusercontent.com/32988384/213200570-e48acaff-289e-4a56-b32a-b061dd4cec85.png)

Now:
![image](https://user-images.githubusercontent.com/32988384/213200668-6e8b652e-255f-4cae-815e-25b06e65cf05.png)

Since it can be used in the context menu, we can keep it open for a while
And it's better if it doesn't block the view by default

Reply:
I made it a bit wider eventually, but not as wide as it used to be